packages/eslint-plugin/docs/rules/ban-types.mdx

```diff
@@ -74,9 +68,6 @@ The default options provide a set of "best practices", intended to provide safet
 - Avoid the `Function` type, as it provides little safety for the following reasons:
   - It provides no type safety when calling the value, which means it's easy to provide the wrong arguments.
   - It accepts class declarations, which will fail when called, as they are called without the `new` keyword.
-- Avoid the `Object` and `{}` types, as they mean "any non-nullish value".
```

Comment:
We still need to discuss `Object` separately because it's not an "uppercase primitive type".

Reply:
i think the thought here was to keep it simple and sweep that caveat under the rug a bit: https://github.com/typescript-eslint/typescript-eslint/pull/8977/files#r1576801720

Reply:
Yeah, given how rare `Object` is in the wild I think we're safe to sweep?

Reply:
Like, @Josh-Cena I actually thought you were in favor of sweeping the `Object` stuff under the rug - maybe I misinterpreted your messages in our private channel. For this discussion's visibility, what do you see as the reason why we wouldn't want to do this?

Reply:
I'm not sure what you mean by "sweeping under the rug". Like, we still have logic in place to handle it, and we have error messages for it. All I'm saying is, if you remove this sentence, then it reads to me as "`Object` will not be handled", because the statement "Don't use the upper-case primitive types, you should use the lower-case types for consistency." does not encompass `Object`: `Object` is not an "upper-case primitive type". You can say "Don't use the upper-case primitive types or `Object`", maybe.

Reply:
Oh! Ok phew yes 😄 I think I was tired and getting confused. Yes, that makes a lot of sense. 👍 will do!